src/Storage/Masa.Contrib.Storage.ObjectStorage.Aliyun/DefaultCredentialProvider.cs

```diff
@@ -43,6 +43,13 @@ public class DefaultCredentialProvider : ICredentialProvider
         ILogger<DefaultCredentialProvider>? logger)
         : this(ossClientFactory, GetAliyunStorageOptions(options.CurrentValue), cache, logger)
     {
+        options.OnChange(aliyunStorageConfigureOptions =>
+        {
+            _options = GetAliyunStorageOptions(aliyunStorageConfigureOptions);
+            SupportSts = !string.IsNullOrEmpty(_options.Sts.RegionId) &&
```

Comment:
It's not support, it should be `IncompleteStsOptions`.

Reply:
processed